src/main/java/com/devathon/preguntonic/services/WebSockerService.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Copyright (c) 2024 Devathon., All Rights Reserved.
+ *
+ */
+package com.devathon.preguntonic.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+@EnableScheduling
+public class WebSockerService {
```

Comment:
Typo
```suggestion
public class WebSocketService {
```

Reply:
I have to change that using vscode (its needed to change file name)